php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -253,6 +253,8 @@ abstract class NDB_BVL_Instrument extends NDB_Page
 
         // Sets up page variables such as $this->commentID and $this->form
         $obj->setup($commentID, $page);
+        // Set page name to testName
+        $obj->name = $instrument;
```

Comment:
I think this should go before the setup call in case the setup function intentionally set it to something.

Reply:
The setup function being used here only sets the `commentID` and `page` values, so it should have no effect on the `name` value. I can make this change (and test it) anyway though 👍